### ISSUE TITLE
Detected an updated version but unable to install it.

### ISSUE THREAD

**Issue by enopuente** [ahmadnassri/node-updated#7]
`npx updated` detected an outdated package: `UPDATED  	 ^7.0.8 → 7.0.9 	 postcss @ ^7.0.8`

But after updating the version, `npm i` wasn't able to install it:
```npm ERR! code ETARGET
npm ERR! notarget No matching version found for postcss@^7.0.9
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of '@telus/kds'
npm ERR! notarget```

**Comment by ruxandrafed** [comment]
Thanks for logging @enopuente! Will look into this!

**Comment by ruxandrafed** [comment]
My first guess was that the `npm` version one has locally could potentially influence what info/access one gets to a particular pkg version from the NPM registry (if you mark a package to only support a particular node version or higher, it could make sense that an older npm/node installed locally wouldn't retrieve the versions with that change included, maybe?).

In any case, @enopuente and I confirmed this was just a temporary NPM registry issue. He was able to eventually install the new version. Closing.
